tests/system/feature_store/test_feature_store.py

```diff
@@ -4818,6 +4820,45 @@ def test_merge_different_number_of_entities(self):
         ).to_dataframe()
         assert_frame_equal(expected_all, df, check_dtype=False)
 
+    @pytest.mark.parametrize("local", [True, False])
+    def test_attributes_in_offline_target(self, local):
+        config_parameters = {} if local else {"image": "mlrun/mlrun"}
+        run_config = fstore.RunConfig(local=local, **config_parameters)
+
+        parquet_path = os.path.relpath(str(self.assets_path / "testdata.parquet"))
+        df = pd.read_parquet(parquet_path)
+
+        run_uuid = uuid.uuid4()
+        v3io_parquet_source_path = f"v3io:///projects/{self.project_name}/df_attributes_source_{run_uuid}.parquet"
+        v3io_parquet_target_path = (
+            f"v3io:///projects/{self.project_name}/df_attributes_target_{run_uuid}"
+        )
+        df.to_parquet(v3io_parquet_source_path)
+
+        feature_set = fstore.FeatureSet(
+            "attributes_fs",
+            entities=[fstore.Entity("patient_id")],
+        )
+
+        offline_target = ParquetTarget(
+            name="test_target",
+            path=v3io_parquet_target_path,
+            attributes={"test_key": "test_value"},
+        )
+        online_target = NoSqlTarget(
+            "no_sql_target", attributes={"test_key_online": "test_value_online"}
+        )
+        source = ParquetSource("test_source", path=v3io_parquet_source_path)
+        feature_set.ingest(
+            source=source,
+            targets=[offline_target, online_target],
+            run_config=run_config,
+        )
+        result_offline_target = get_offline_target(feature_set)
+        result_online_target = get_online_target(feature_set)
```

Comment:
This just demonstrates that the attributes are still in the feature set object. What we actually want to test is that they are stored in the database and retrieved successfully.

From a quick debugger session, it actually seems like that's not happening.

Reply:
It is saved in `fs.status.targets`, not in `fs.spec.targets.`
 I also added a part to load it from the run_db to test.

Does this answer your concern?